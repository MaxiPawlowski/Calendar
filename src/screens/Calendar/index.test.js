import React from 'react';
import { shallow } from 'enzyme';
import { StyledApp as Calendar } from "./index";

describe('App', () => {
  let wrapper;

  describe('when rendered', () => {
    beforeAll(() => {
      wrapper = shallow(
        <Calendar
          reminders={[]}
          addReminder={()=>{}}
        />).dive();
    });

    it('should render correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
