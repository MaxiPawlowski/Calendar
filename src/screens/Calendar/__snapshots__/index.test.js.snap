// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App when rendered should render correctly 1`] = `
<div
  className="Calendar-root-1"
>
  <div
    className="Calendar-sidebar-2"
  >
    <WithStyles(ReminderForm)
      handleSave={[Function]}
    />
    <MuiPickersUtilsProvider
      utils={[Function]}
    >
      <PickerWithState
        allowKeyboardControl={true}
        invalidDateMessage="Invalid Date Format"
        label="Calendar Date"
        maxDate={2100-01-01T00:00:00.000Z}
        maxDateMessage="Date should not be after maximal date"
        minDate={1900-01-01T00:00:00.000Z}
        minDateMessage="Date should not be before minimal date"
        onChange={[Function]}
        openTo="year"
        value={2019-10-01T03:00:00.000Z}
        views={
          Array [
            "year",
            "month",
          ]
        }
      />
    </MuiPickersUtilsProvider>
  </div>
  <WithStyles(ReminderForm)
    calendarDate={
      Object {
        "month": 10,
        "year": 2019,
      }
    }
    deleteReminder={[Function]}
    handleDayExpand={[Function]}
    reminders={Array []}
  />
  <WithStyles(RemindersDialog)
    data={
      Object {
        "date": 2017-02-14T12:51:48.000Z,
        "reminders": Array [],
        "weathers": Array [],
      }
    }
    deleteReminder={[Function]}
    editReminder={[Function]}
    getWeatherFromReminders={[Function]}
    handleClose={[Function]}
    open={false}
  />
</div>
`;
